index.html

```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
                   </ul>
                 </li>
                 <li class="frameworks">
-                  <p class="">Frameworks</p>
+                  <h2> Frameworks </h2>
                 </li>
                 <li class="skills">
                   <h2> Skills </h2>
                 </li>
               </ul>
             </div>
-    </section>
-
-
+          </section>
 
+          <section class="contact__me__section">
```

Comment:
- [ ] Nice working on your form it looks good and works well, just a little change to make it look like what we have in the Figma template, kindly add more space above the title.
![design](https://user-images.githubusercontent.com/48371156/165289802-29b28ee2-8fc8-4525-bafc-1c8f26a3115b.jpg)